tool/zjit_bisect.rb

```diff
@@ -1,12 +1,26 @@
 #!/usr/bin/env ruby
 require 'logger'
 require 'open3'
+require 'optparse'
+require 'shellwords'
 require 'tempfile'
 require 'timeout'
 
-RUBY = ARGV[0] || raise("Usage: ruby jit_bisect.rb <path_to_ruby> <options>")
-OPTIONS = ARGV[1] || raise("Usage: ruby jit_bisect.rb <path_to_ruby> <options>")
-TIMEOUT_SEC = 5
+ARGS = {:timeout => 5}
+OptionParser.new do |opts|
+  opts.banner += " <path_to_ruby> -- <options>"
+  opts.on("--timeout=TIMEOUT_SEC", "Seconds until child process is killed") do |timeout|
+    ARGS[:timeout] = Integer(timeout)
+  end
+  opts.on("-h", "--help", "Prints this help") do
+    puts opts
+    exit
+  end
+end.parse!
+
+RUBY = ARGV[0] || raise("Usage: ruby jit_bisect.rb <path_to_ruby> -- <options>")
```

Comment:
WDYT about using `RbConfig.ruby` to get the current ruby path? Feels easier to use, and we can have an option to pass  a path too.

Reply:
Eh, prefer explicit here